docs/design/coreclr/jit/ryujit-overview.md

```diff
@@ -208,20 +209,25 @@ The top-level function of interest is `Compiler::compCompile`. It invokes the fo
 | [Inlining](#inlining) | The IR for inlined methods is incorporated into the flowgraph. |
 | [Struct Promotion](#struct-promotion) | New lclVars are created for each field of a promoted struct. |
 | [Mark Address-Exposed Locals](#mark-addr-exposed) | lclVars with references occurring in an address-taken context are marked.  This must be kept up-to-date. |
+| Early liveness | Compute lclVar liveness for use by phases up to and including global morph. |
+| Forward Subtitution | Eliminate SDSU-like locals by substituting their values directly into uses. |
+| Physical promotion | Split struct locals into primitives based on access patterns. |
```

Comment:
These and other new phases are looking for a writer to expand on what they do.

Reply:
In particular, we are missing any mention of PGO-related phases in the overview.